src/create.rs

```diff
@@ -308,6 +310,16 @@ impl<'a> Builder<'a> {
         self
     }
 
+    /// Sets the variant.
```

Comment:
[Variants](https://github.com/mmstick/cargo-deb/pull/61) in the [cargo-deb](https://github.com/mmstick/cargo-deb) project are for defining different configurations of the `cargo deb` command within a project's `Cargo.toml` file. It is not for selecting a package within a workspace. The doc-comments do not match the intention of variants.

It appears possible to define different installers for different packages within a Workspace using variants, but the argument does not select the specific package.